### PR TITLE
CalibCalorimetry/CaloTPG :formatting fix for gcc 6.0 misleading-indentation warning

### PR DESCRIPTION
**Pull request by gartung**
/build/cmsbuild/jenkins-workarea/workspace/build-any-ib/w/tmp/BUILDROOT/8258e37281729e179b5af9c3b033d930/opt/cmssw/slc6_amd64_gcc600/cms/cmssw/CMSSW_8_1_X_2016-06-26-2300/src/CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.cc:59:4: warning: this 'if' clause does not guard... [-Wmisleading-indentation]
     if ((not HcalGenericDetId(*i).isHcalDetId()) and
    ^~
/build/cmsbuild/jenkins-workarea/workspace/build-any-ib/w/tmp/BUILDROOT/8258e37281729e179b5af9c3b033d930/opt/cmssw/slc6_amd64_gcc600/cms/cmssw/CMSSW_8_1_X_2016-06-26-2300/src/CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.cc:63:6: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the 'if'
      continue;
      ^~~~~~~~

### CODE REVIEW

**Comment by mmusich** [comment]
please test

**Comment by ghellwig** [comment]
@gartung can you clarify the intention of splitting the change into two commits?
Do want to point out that someone might have forgotten the curly braces when writing the code?

**Comment by gartung** [comment]
My initial commit added the curly braces, but I did not want to change the meaning of the code.

**Comment by ghellwig** [comment]
+1
the last commit to this file (8b8909957f8e7e2d99992087483b106ff7e2b6b8) clearly suggests that this was just an accidental identation

File: CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.cc [hunk 1]
```diff
@@ -60,7 +60,7 @@ void CaloTPGTranscoderULUT::loadHCALCompress(HcalLutMetadata const& lutMetadata,
 	      (not HcalGenericDetId(*i).isHcalZDCDetId()) and
 	      (not HcalGenericDetId(*i).isHcalCastorDetId()))
 	    edm::LogWarning("CaloTPGTranscoderULUT") << "Encountered invalid HcalDetId " << HcalGenericDetId(*i);
-	    continue;
```
Comment (gartung) [inline CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.cc]: Is this continue meant to be in the if block started on line 59?

Comment (ghellwig) [inline CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.cc]: @gartung I think not, as I said above. Maybe @bsunanda (who seems to be the last contributor?) can comment?

**Pull request merged**